ow_materials/src/MaterialDistributionPlugin.cpp

```diff
@@ -160,19 +361,17 @@ void MaterialDistributionPlugin::handleCollisionBulk(MaterialBlend const &blend)
 Color MaterialDistributionPlugin::interpolateColor(
   MaterialBlend const &blend) const
 {
-  return std::accumulate(blend.getBlendMap().begin(), blend.getBlendMap().end(),
-    Color({0.0, 0.0, 0.0}),
+  return std::accumulate(
+    blend.getBlendMap().begin(), blend.getBlendMap().end(),
+    Color{0.0, 0.0, 0.0},
     [this]
     (Color value, MaterialBlend::BlendType::value_type const &p) {
       const auto m = m_material_db->getMaterial(p.first);
-      // TODO: define + and * operators for color
-      return Color(
-        {
-          value.r + p.second * m.color.r,
-          value.g + p.second * m.color.g,
-          value.b + p.second * m.color.b
-        }
-      );
+      return Color {
+        value.r + p.second * (m.visualize_color.r - value.r),
```

Comment:
ignition::math::Color has arithmetic operators that would simplify this code.

Reply:
Great to know! I nearly went the route of implementing my own. I think I am now sold on switching to ignition::math::Color